crates/autopilot/src/driver_model.rs

```diff
@@ -0,0 +1,171 @@
+//! Types for communicating with drivers as defined in
```

Comment:
Why do we need these types here? The autopilot already knows how to talk to a driver. :thinking:

Reply:
How does it enter here? I must have overlooked it with so many changes, sorry.

Reply:
I think it was because of my latest rebase.